manual/freemake-video-converter/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,15 @@
+$uninstaller = '\Freemake\Freemake Video Converter\Uninstall\unins000.exe'
+if ("${Env:ProgramFiles(x86)}") {
+  $fullPath = "${Env:ProgramFiles(x86)}$uninstaller"
+} else {
+  $fullPath = "${Env:ProgramFiles}$uninstaller"
+}
+
+$packageArgs = @{
+  packageName = 'freemake-video-converter'
+  fileType    = 'exe'
+  silentArgs  = '/VERYSILENT /NORESTART'
+  file        = $fullPath
```

Comment:
Since the directory can be overridden on install by users passing the right arguments through, there is no guarantee where this is installed. As @majkinetor mentioned, using the chocolatey-core extension will look in the right places for that installation path.